frontend/public/components/storage/access-modes/access-mode.tsx

```diff
@@ -78,10 +67,11 @@ export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) =>
 
   return (
     <FormGroup
-      label={t('console-app~Access Mode')}
+      label={label}
       isRequired
       fieldId="access-mode"
       className={formClassName}
+      htmlFor="access-mode"
```

Comment:
I'm a little unsure if this is right. Is there an element with an `access-mode` ID?